tests/bookmarks_and_history/test_opened_website_in_new_window_present_in_toolbar_history.py

```diff
@@ -0,0 +1,40 @@
+from selenium.webdriver import Firefox
+
+from modules.browser_object_panel_ui import PanelUi
+from modules.browser_object_tabbar import TabBar
+from modules.page_object_generics import GenericPage
+
+FACEBOOK_URL = "https://www.facebook.com/"
+AMAZON_URL = "https://www.amazon.com/"
+YOUTUBE_URL = "https://www.youtube.com/"
+
+WEBSITES = [FACEBOOK_URL, AMAZON_URL]
+
+
+def test_the_website_opened_in_new_window_is_present_in_history_menu(driver: Firefox):
+    """
+    C118805 - Verify that the website opened in new window is displayed in the Toolbar History submenu on top of the
+    list
+    """
+
+    for url in WEBSITES:
+        GenericPage(driver, url=url).open()
+
+    tabs = TabBar(driver)
+    panel_ui = PanelUi(driver)
+
+    panel_ui.open_panel_menu()
+    panel_ui.select_panel_setting("panel-ui-new-window")
+    tabs.wait_for_num_tabs(2)
+    driver.switch_to.window(driver.window_handles[-1])
```

Comment:
This can be substituted by an existing method `tabs.switch_to_new_window()`, up to you if you want to change it to that tho